src/contrib/php_array_api.h

```diff
@@ -350,7 +350,7 @@ char *php_array_zval_to_string(zval *z, int *plen, zend_bool *pfree) {
 			zval c = *z;
 			zval_copy_ctor(&c);
 			convert_to_string(&c);
-			*pfree = ! IS_INTERNED(Z_STR(c));
```

Comment:
For context, this was removed in https://github.com/php/php-src/commit/64e2832bc8065f24c4d273075608f33826c71fd8